OpenSim/Simulation/Model/GeometryPath.h

```diff
@@ -57,6 +57,12 @@ class PointForceDirection;
  */
 class OSIMSIMULATION_API GeometryPath : public ModelComponent {
 OpenSim_DECLARE_CONCRETE_OBJECT(GeometryPath, ModelComponent);
+    //=============================================================================
+    // OUTPUTS
+    //=============================================================================
+    OpenSim_DECLARE_OUTPUT(length, double, getLength, SimTK::Stage::Position);
+    // 
+    OpenSim_DECLARE_OUTPUT(speed, double, getLengtheningSpeed, SimTK::Stage::Velocity);
```

Comment:
I prefer lengthening_speed, for consistency with PathSpring, and also because it's reasonable for people to think it's shortening speed otherwise.